app/components/cluster-dashboard/component.js

```diff
@@ -28,7 +30,7 @@ export default Component.extend({
   }),
 
   inactiveNodes: computed('nodes.@each.state', function() {
-    return get(this, 'nodes').filter( (n) => get(n, 'state') !== 'active' && get(n, 'state') !== 'cordoned' );
+    return get(this, 'nodes').filter( (n) => get(n, 'state') !== 'active' && HEALTHY_STATES.indexOf(get(n, 'state')) === -1 );
```

Comment:
Is there a reason we are not using 'C.ACTIVEISH_STATES` for this? Do we only need to check active?